tests/vmemcache_test_mt.c

```diff
@@ -43,6 +43,7 @@
 #include "os_thread.h"
 
 #define BUF_SIZE 256
+#define MAX_PUTS 25000 /* maximum number of puts */
```

Comment:
this should be a variable (or a multipler) based on `ops_count`. Right now, you shouldn't change one without changing another.